src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.ConvertToString.liquid

```diff
@@ -26,7 +26,8 @@ private string ConvertToString(object value, System.Globalization.CultureInfo cu
                 }
             }
 
-            return System.Convert.ToString(System.Convert.ChangeType(value, System.Enum.GetUnderlyingType(value.GetType()), cultureInfo));
+            var converted = System.Convert.ToString(System.Convert.ChangeType(value, System.Enum.GetUnderlyingType(value.GetType()), cultureInfo));
+            return (converted is null) ? string.Empty : converted;
```

Comment:
I think we should use 

    return converted == null ? string.Empty : converted;

to keep compatibility with older C# versions.

Reply:
I did it in that style because the existing code in the template is like that:
https://github.com/RicoSuter/NSwag/blob/285e1c09c8f3fa17dd435be469ec2a046a4e0895/src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.ConvertToString.liquid#L47

Will change if you want though.